app/services/irs_attempts_api/redis_client.rb

```diff
@@ -0,0 +1,39 @@
+module IrsAttemptsApi
+  class RedisClient
+    cattr_accessor :redis_pool do
+      ConnectionPool.new(size: 10) do
+        Redis::Namespace.new(
+          'irs-attempt-api',
+          redis: Redis.new(url: IdentityConfig.store.redis_irs_attempt_api_url),
+        )
+      end
+    end
+
+    def write_event(jti, jwe)
```

Comment:
let's make these keyword arguments so we minimize the chances of them being swapped?
```suggestion
    def write_event(jti:, jwe:)
```